docs/Inleiding.md

```diff
@@ -0,0 +1,43 @@
+# Inleiding
+
+## Historie
+
+VNGRealisatie is enkele jaren geleden begonnen met het inzetten van Imvertor als tool om de kwaliteit en de consistentie van de informatiemodellen te controleren. Tijdens dit traject rol ontstond het idee om op basis het informatiemodel en een omzettings-algoritme de basisentiteiten voor StUF te genereren. Toen dit realiseerbaar bleek te zijn bleek dat daar veel consistentie-voordelen aan zaten. Om die voordelen door te trekken naar de daadwerkelijke koppelvlakken is vervolgens ook een methodiek ontwikkeld om de koppelvlakken zelf te modelleren en genereren.
```

Comment:
Wijzig

> Tijdens dit traject rol ontstond het idee om op basis het informatiemodel en een omzettings-algoritme de basisentiteiten voor StUF te genereren.

in

_Tijdens dit traject ontstond het idee om op basis van het informatiemodel en een omzettings-algoritme de basisentiteiten voor StUF te genereren._

Reply:
aangepast